src/Microsoft.DocAsCode.Build.TableOfContents/TocDocumentProcessor.cs

```diff
@@ -165,34 +169,34 @@ private void RegisterTocMap(TocItemViewModel item, string key, IDocumentBuildCon
             }
         }
 
-        private string ResolveHref(string originalPathToFile, FileModel model, IDocumentBuildContext context)
+        private string ResolveHref(string pathToFile, string originalPathToFile, FileModel model, IDocumentBuildContext context)
         {
-            if (!Utility.IsSupportedRelativeHref(originalPathToFile))
+            if (!Utility.IsSupportedRelativeHref(pathToFile))
             {
-                return originalPathToFile;
+                return pathToFile;
             }
 
-            var index = originalPathToFile.IndexOf('#');
+            var index = pathToFile.IndexOf('#');
             if (index == 0)
             {
-                throw new DocumentException($"Invalid toc link: {originalPathToFile}.");
+                throw new DocumentException($"Invalid toc link: {pathToFile}.");
```

Comment:
OK. seems relative path is clearer.

---
In reply to: [75433940](https://github.com/dotnet/docfx/pull/623#discussion_r75433940) [](ancestors = 75433940)